### ISSUE TITLE
unet loss error

### ISSUE THREAD

**Issue by lfdeep** [meetshah1995/pytorch-semseg#159]
When I trained unet in the pascal voc dataset and i set learning rate=0.001, the loss was particularly large:

Iter [1833/300000]  Loss: 115374464.0000  Time/Image: 0.0173
Iter [1834/300000]  Loss: 22800710.0000  Time/Image: 0.0163
Iter [1835/300000]  Loss: 470180.6250  Time/Image: 0.0180
Iter [1836/300000]  Loss: 32794706.0000  Time/Image: 0.0177
Iter [1837/300000]  Loss: 10254959.0000  Time/Image: 0.0160
Iter [1838/300000]  Loss: 127539232.0000  Time/Image: 0.0158

i don't know how to solve it!

**Comment by Qirui-Y** [comment]
Hello, can you share your config file?

**Comment by flyking1994** [comment]
Have you solve your problem?

**Comment by GewelsJI** [comment]
learning rate is too large, plz try small lr
